.storybook/main.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  stories: ['../pages/**/*.stories.tsx'],
+  addons: ['@storybook/addon-storysource'],
```

Comment:
After you start to use `addon-docs`, you will be no need to use `storysource` as well.